src/patternfly/components/ClipboardCopy/examples/clipboard-copy-expansion-example.hbs

```diff
@@ -60,3 +58,31 @@ Read-Only
     This is an editable version of the Copy to Clipboard Component that has an expandable section. Got a lot of text here, need to see all of it? Click that arrow on the left side and check out the resulting expansion.
   {{/clipboard-copy-expandable-content}}
 {{/clipboard-copy}}
+<br>
+<br>
+JSON Object (wrap code in pre)
+{{#> clipboard-copy clipboard-copy--id="7" clipboard-copy--IsExpanded="true"}}
+	{{#> clipboard-copy-group}}
+    {{#> clipboard-copy-group-toggle clipboard-copy-group-toggle--attribute=(concat 'id="toggle-' clipboard-copy--id '" aria-labelledby="toggle-' clipboard-copy--id ' text-input-' clipboard-copy--id '" aria-controls="content-' clipboard-copy--id '"')}}
+    {{/clipboard-copy-group-toggle}}
+    {{#> form-control controlType="input" form-control--attribute=(concat 'readonly type="text" value="Code goes here" id="text-input-' clipboard-copy--id '" aria-label="Copyable input"')}}
+	  {{/form-control}}
+	  {{#> clipboard-copy-group-copy clipboard-copy-group-copy--attribute=(concat 'id="copy-button-' clipboard-copy--id '" aria-labelledby="copy-button-' clipboard-copy--id ' text-input-' clipboard-copy--id '"')}}
+    {{/clipboard-copy-group-copy}}
+  {{/clipboard-copy-group}}
+  {{#> clipboard-copy-expandable-content clipboard-copy-expandable-content--attribute=(concat 'id="content-' clipboard-copy--id '"')}}
+<pre>
```

Comment:
nit - can we indent this?

Reply:
@mcoker only problem is that when I indent it it adds spaces, I tried converting the "spaces" to "tabs" in visual studio but that also didn't work. Is there a way where we could full-proof this?

<img width="887" alt="Screen Shot 2019-09-16 at 11 18 04 AM" src="https://user-images.githubusercontent.com/20118816/64970547-0698e480-d874-11e9-9b00-c660f7174866.png">

Reply:
Oh! Makes perfect sense, please disregard my comment, this is fine :)

Reply:
Although this file has too many spaces for tabs now. Can we make sure the tabs are set to 2 spaces? https://github.com/patternfly/patternfly-next/blob/172a1c8c92de0dd34eac93398d58afbabf4e6434/src/patternfly/components/ClipboardCopy/examples/clipboard-copy-expansion-example.hbs